ios/FluentUI/Badge Field/BadgeViewTokenSet.swift

```diff
@@ -0,0 +1,201 @@
+                    case .severeWarning:
+                        return theme.aliasTokens.colors[.severeForeground1]
+                    case .warning:
+                        return theme.aliasTokens.colors[.warningForeground1]
+                    case .success:
+                        return theme.aliasTokens.colors[.successForeground1]
+                    case .neutral:
+                        return theme.aliasTokens.colors[.foreground2]
+                    }
+                }
+            case .foregroundFilledColor:
+                return .dynamicColor {
+                    switch style() {
+                    case .default:
+                        return theme.aliasTokens.colors[.foregroundOnColor]
+                    case .danger, .severeWarning, .success:
+                        return theme.aliasTokens.colors[.foregroundLightStatic]
+                    case .warning:
+                        return theme.aliasTokens.colors[.foregroundDarkStatic]
+                    case .neutral:
+                        return theme.aliasTokens.colors[.foreground1]
+                    }
+                }
+            case .foregroundDisabledColor:
+                return .dynamicColor {
+                    switch style() {
+                    case .default:
+                        return theme.aliasTokens.colors[.brandForegroundDisabled1]
+                    case .danger, .severeWarning, .warning, .success, .neutral:
+                        return theme.aliasTokens.colors[.foregroundDisabled1]
+                    }
+                }
+            case .borderRadius:
+                return .float({
+                    switch size() {
+                    case .small:
+                        return GlobalTokens.corner(.radius20)
+                    case .medium:
+                        return GlobalTokens.corner(.radius40)
+                    }
+                })
+            case .textFont:
+                return .fontInfo({
+                    switch size() {
+                    case .small:
+                        return theme.aliasTokens.typography[.caption1]
+                    case .medium:
+                        return theme.aliasTokens.typography[.body2]
+                    }
+                })
+            }
+        }
+    }
+
+    /// Defines the style of the Badge.
+    var style: () -> Style
+
+    /// Defines the size of the Badge.
+    var size: () -> Size
+}
+
+// MARK: Constants
+extension BadgeViewTokenSet {
+    static func horizontalPadding(_ size: Size) -> CGFloat {
+        switch size {
+        case .small:
+            return GlobalTokens.spacing(.size40)
+        case .medium:
+            return GlobalTokens.spacing(.size80)
+        }
+    }
+
+    static func verticalPadding(_ size: Size) -> CGFloat {
+        switch size {
+        case .small:
+            return 1.5
+        case .medium:
+            return GlobalTokens.spacing(.size40)
+        }
```

Comment:
Spoke some offline, but we shouldn't need hard coded values like this anymore, generally. Looks like in this specific case it can also just be `GlobalTokens.spacing(.size40)`

Reply:
You mean `GlobalTokens.spacing(.size20)` right?